1-uagents/finance/company-ticker-resolver-agent/.avctl/config.toml

```diff
@@ -0,0 +1,3 @@
+[agent]
+  address = ""
```

Comment:
Why is this empty (similarly some of the below)? Are they not deployed?

Reply:
That's right, not deployed yet.